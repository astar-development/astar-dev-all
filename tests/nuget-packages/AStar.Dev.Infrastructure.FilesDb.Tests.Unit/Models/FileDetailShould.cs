namespace AStar.Dev.Infrastructure.FilesDb.Models;

public sealed class FileDetailShould
{
    [Fact]
    public void ReturnTheExpectedToStringRepresentation()
    {
        var fileDetail = new FileDetail();

        fileDetail.ToString().ShouldMatchApproved();
    }
}
