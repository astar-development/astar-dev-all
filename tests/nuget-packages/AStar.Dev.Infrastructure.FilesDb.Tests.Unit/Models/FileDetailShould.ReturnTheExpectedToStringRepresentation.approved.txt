{
  "fileDetailClassifications": [],
  "id": 0,
  "fileAccessDetail": {
    "id": 0,
    "detailsLastUpdated": null,
    "lastViewed": null,
    "softDeleted": false,
    "softDeletePending": false,
    "moveRequired": false,
    "hardDeletePending": false
  },
  "fileName": "",
  "directoryName": "",
  "fullNameWithPath": "",
  "height": 0,
  "width": 0,
  "fileSize": 0,
  "isImage": false
}